src/roles/vsd-upgrade-inplace/tasks/main.yml

```diff
@@ -36,3 +59,4 @@
     tasks_from: monit_status
   vars:
     no_report: "True"
+#commnet
```

Comment:
Why is this line here?

Reply:
it was a mistake. I was explaining to Arogya How the git works, I mistakenly committed the changes in the wrong repo

Reply:
it's removed